2020/8xxx/CVE-2020-8555.json

```
@@ -1,18 +1,216 @@
 {
-    "data_type": "CVE",
-    "data_format": "MITRE",
-    "data_version": "4.0",
-    "CVE_data_meta": {
-        "ID": "CVE-2020-8555",
-        "ASSIGNER": "cve@mitre.org",
-        "STATE": "RESERVED"
-    },
-    "description": {
-        "description_data": [
-            {
-                "lang": "eng",
-                "value": "** RESERVED ** This candidate has been reserved by an organization or individual that will use it when announcing a new security problem. When the candidate has been publicized, the details for this candidate will be provided."
-            }
+  "data_type": "CVE",
+  "data_format": "MITRE",
+  "data_version": "4.0",
+  "generator": {
+    "engine": "Vulnogram 0.0.9"
+  },
+  "CVE_data_meta": {
+    "ID": "CVE-2020-8555",
+    "ASSIGNER": "security@kubernetes.io",
+    "DATE_PUBLIC": "",
+    "TITLE": "Kubernetes  kube-controller-manager SSRF",
+    "AKA": "",
+    "STATE": "PUBLIC"
+  },
+  "source": {
+    "defect": [
+      "https://github.com/kubernetes/kubernetes/issues/91542"
+    ],
+    "advisory": "",
+    "discovery": "EXTERNAL"
+  },
+  "affects": {
+    "vendor": {
+      "vendor_data": [
+        {
+          "vendor_name": "Kubernetes",
+          "product": {
+            "product_data": [
+              {
+                "product_name": "Kubernetes",
+                "version": {
+                  "version_data": [
+                    {
+                      "version_name": "",
+                      "version_affected": "",
+                      "version_value": "prior to 1.15.12",
```

normally this is more like this:
```suggestion
                      "version_name": "",
                      "version_affected": "<",
                      "version_value": "1.15.12",
```